docs/reference/cli/rack.md

```diff
@@ -27,18 +27,35 @@ Get information about the rack
 
 Install a new Rack
 
+> note: To install the rack into the console with the specified runtime, region, and optional [parameters](https://docs.convox.com/configuration/rack-parameters/): provide orgname/rackname in place of `<name>`
+
 ### Usage
 ```html
     convox rack install <provider> <name> [option=value]...
+
+    convox rack install <provider> <orgname>/<rackname> region=<region> --runtime=<runtime-id> [option=value]...
 ```
+
+flags:
+  - `runtime`: runtime integration ID 
+  - `version`: specify the rack version to install
+
+> note: To install a rack into an organization with runtime integration, ensure your CLI is updated to the latest version. For detailed instructions on updating CLI, please see [CLI Management](https://docs.convox.com/management/cli-rack-management/).
+
+> note: Obtain the runtime ID by running `convox runtimes <orgname>`.
+
 ### Examples
 ```html
     $ convox rack install local dev
     ...
 
     $ convox rack install aws production region=eu-west-1 node_type=t3.large
     ...
+
+    $ convox rack install aws my-org/staging region=us-east-1 --runtime=20e58437-fab7-4124-aa5a-2e5978f1149e
+    ...
 ```
+
```

Comment:
why these changes are in this pr?

Reply:
I created the fix branch from master, staging did not had this docs change.